cinema/models.py

```diff
@@ -35,12 +39,20 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def convert_uploded_image_name(instance: "Movie", filename: str) -> str:
```

Comment:
There is a typo in the function name `convert_uploded_image_name`. It should be `convert_uploaded_image_name` to maintain consistency and avoid potential errors.